### ISSUE TITLE
guide

### ISSUE THREAD

**Issue by Leuhan76** [phhusson/treble_experimentations#1862]
hello, i would like to know how i have to flash aosp using fastboot
a detailed guide if possible
(I am new at this)

**Comment by Merat70** [comment]
What is the brand and model of your phone?

**Comment by haruki-taka8** [comment]
You can find a device-specific guide in the [Wiki](https://github.com/phhusson/treble_experimentations/wiki) of this repository.

Generally speaking, flashing `system` and/or `vbmeta` will suffice.

**Comment by Leuhan76** [comment]
I have an alcatel 3x 2019 (5048a) I do not know if it will be compatible with this rom or I do not know how functional it will be and I want to find out everything very well so as not to run out of phone

**Comment by Leuhan76** [comment]
> Puede encontrar una guía específica del dispositivo en la [Wiki](https://github.com/phhusson/treble_experimentations/wiki) de este repositorio.
> 
> En general, parpadear `system`y / o `vbmeta`bastará.

My device is not on that list, does that mean it is not compatible?

**Comment by haruki-taka8** [comment]
> supports project treble

If it supports Project Treble, theoretically you can boot a GSI with it. Your mileage may differ though.

From what I can find about your device (very limited!), you can install it using TWRP or fastboot. It seems there is also Secure Boot, so you might need to flash vbmeta as well.

**Comment by Leuhan76** [comment]
> > supports project treble
> 
> If it supports Project Treble, theoretically you can boot a GSI with it. Your mileage may differ though.
> 
> From what I can find about your device (very limited!), you can install it using fastboot. It seems there is also Secure Boot, so you might need to flash vbmeta as well.

If they told me the same thing and that I need twrp because On the device there are several partitions In order to delete the "system" and "data" partitions, I need twrp but for my device there is no

**Comment by haruki-taka8** [comment]
Do not delete system and data partition! All you need is to flash the required images (system and vbmeta) using fastboot, and then wipe the userdata partition.

**Comment by Leuhan76** [comment]
These, I asked that person for sure because I need to do that and he told me that there are several partitions on the device and that In order to delete the "system" and "data" partitions, I need something that can do it, I mean twrp

**Comment by haruki-taka8** [comment]
Why do you need to delete "system" and "data"?

**Comment by Leuhan76** [comment]
To be able to install the gsi (ROM) without problems
at least that's what he told me

> Why do you need to delete "system" and "data"?

**Comment by haruki-taka8** [comment]
I don't believe you can delete partitions with TWRP as well. You can at most wipe it, which you can also do with fastboot.

**Comment by Leuhan76** [comment]
> I don't believe you can delete partitions with TWRP as well. You can at most wipe it, which you can also do with fastboot.

so I just flash the img and that's it?

**Comment by haruki-taka8** [comment]
Most likely yes! Try just flashing the `img` and boot it.

**Comment by Leuhan76** [comment]
But don't I have to clear the data cache before and after?

> Most likely yes! Try just flashing the `img` and boot it.

**Comment by haruki-taka8** [comment]
You only need to clear them afterwards.
